src/pygeoprocessing/routing/routing.pyx

```diff
@@ -3039,6 +3039,10 @@ def extract_strahler_streams_d8(
             stream segment outlet.
         * "ds_y" (int): the downstream y coordinate in raster space for the
             stream segment outlet.
+        * "ds_x_1" (int): the downstream x coordinate for one pixel upstream
+            from the stream segment outlet raster space coordinates.
+        * "ds_y_1" (int): the downstream y coordinate for one pixel upstream
+            from the stream segment outlet raster space coordinates.

```

Comment:
Ah, sorry, this reads a little ambiguously to me.  How about something like "the {x,y} coordinate (in raster space) that is 1 pixel upstream from the stream segment outlet"?

Reply:
Thanks! I changed all the wording of {ds,us}_{x,y}{,_1} so they sound better in 766bc8f